### PR TITLE
Contract requests fixes

### PR DESCRIPTION
**Pull request by kiurchv**

### CODE REVIEW

File: apps/graphql/test/web/resolvers/contract_request_resolver_test.exs [hunk 1]
```diff
@@ -247,6 +272,31 @@ defmodule GraphQLWeb.ContractRequestResolverTest do
...
+        conn
+        |> put_client_id()
+        |> post_query(@list_query, variables)
+        |> json_response(200)
+
+      resp_entities = get_in(resp_body, ~w(data contractRequests nodes))
+
+      assert nil == resp_body["errors"]
+      assert 1 == length(resp_entities)
+      assert requested_contractor_legal_entity.id == hd(resp_entities)["contractorLegalEntity"]["databaseId"]
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/contract_request_resolver_test.exs]: the same

File: apps/graphql/test/web/resolvers/contract_request_resolver_test.exs [hunk 2]
```diff
@@ -199,6 +202,28 @@ defmodule GraphQLWeb.ContractRequestResolverTest do
...
+
+      resp_body =
+        conn
+        |> put_client_id()
+        |> post_query(@list_query, variables)
+        |> json_response(200)
+
+      resp_entities = get_in(resp_body, ~w(data contractRequests nodes))
+
+      assert nil == resp_body["errors"]
```
Comment (AlexKovalevych) [inline apps/graphql/test/web/resolvers/contract_request_resolver_test.exs]: consider using https://hexdocs.pm/ex_unit/ExUnit.Assertions.html#refute/1

File: apps/graphql/test/web/resolvers/contract_request_resolver_test.exs [hunk 3]
```diff
@@ -199,6 +202,28 @@ defmodule GraphQLWeb.ContractRequestResolverTest do
...
+      assert 1 == length(resp_entities)
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/contract_request_resolver_test.exs]:
```suggestion
      assert [resp_entity] = resp_entities
```

File: apps/graphql/test/web/resolvers/contract_request_resolver_test.exs [hunk 4]
```diff
@@ -199,6 +202,28 @@ defmodule GraphQLWeb.ContractRequestResolverTest do
...
+      assert requested_contract_request.id == hd(resp_entities)["databaseId"]
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/contract_request_resolver_test.exs]:
```suggestion
      assert %{"databaseId" => ^requested_contract_request.id} = resp_entity
```

**Pull request merged**